modules/report/js/commerce_pos_report.js

```diff
@@ -1,135 +0,0 @@
-(function ($) {
```

Comment:
commerce_pos_keypad.js uses Drupal.CommercePosReport.currencyFormat(). That's one example. There are probably more.